src/main/java/uk/ac/ebi/subs/validator/taxon/core/TaxonomyService.java

```diff
@@ -10,7 +10,7 @@
     private final String ENA_TAXONOMY_SERVICE = "http://www.ebi.ac.uk/ena/data/taxonomy/v1/taxon/";
```

Comment:
Do we want to hard code the taxonomy service URL to this class?
Is not it better to externalise it to a config file (properties, yml ...)?